docker-compose.yml

```diff
@@ -74,6 +74,17 @@ services:
       DEBUG: "true"
       # Uncomment the following line for MAX logins
       # MAX_URL: https://login.test.max.gov/cas/login
+    depends_on:
+      - ombpdf-js-bundler
```

Comment:
Note that this is overriding the `depends_on` in `PROD_API`. You'll probably want to add back `persistent_db` and `minio`